### ISSUE TITLE
No ability to disable a signature pad

### ISSUE THREAD

**Issue by josh-thompson13** [YanYuanFE/react-native-signature-canvas#275]
Does anyone know if there is a way to disable the signature pad? I want to 'lock' the pad after a user saves their signature.

**Comment by YanYuanFE** [comment]
This function is not available for the time being. I will consider adding this function in a few days

**Comment by atefwahab** [comment]
> This function is not available for the time being. I will consider adding this function in a few days

hello have you added this ability??

**Comment by kkaplinski** [comment]
@atefwahab @josh-thompson13  I would suggest to just hide the signature pad and show signature in the `<Image>` component or just create overlay with position absolute. Adding some semi transparent overlay also would improve UX and suggest user that signature pad is disabled.

**Comment by josh-thompson13** [comment]
> @atefwahab @josh-thompson13 I would suggest to just hide the signature pad and show signature in the `<Image>` component or just create overlay with position absolute. Adding some semi transparent overlay also would improve UX and suggest user that signature pad is disabled.

Yes I did consider this but it does not suit my use case. I need the drawpad to actually be disabled. :)

**Comment by rvmaher** [comment]
Wrap your component in View or whatever you like and set pointerEvents to none.
